utils/mil_tools/nodes/online_bagger.py

```diff
@@ -148,8 +163,11 @@ def subscribe_loop(self):
             i = i + 1
             if i % 1000 == 0:
                 rospy.logdebug('still subscribing!')
+        rospy.loginfo("Subscribed to {} of {} topics, will try again every {} seconds".format(self.successful_subscription_count, 
+            len(self.subscriber_list), self.resubscribe_period))
+        rospy.Timer(rospy.Duration(self.resubscribe_period), self.subscribe)
 
-    def subscribe(self):
+    def subscribe(self, time_info=None):
```

Comment:
You should cancel the timer if all topics are subscribed to so it can stop trying (it should also print a message when this happens)